docs/source/user/fast.farm/InputFiles.rst

```diff
@@ -139,7 +139,7 @@ documentation for details on the input file at the farm level.
 MoorDyn. 
 
 **WrMooringVis** [swithch]   Write shared mooring line visualization, at
-DT_Mooring timestep
+the globale FAST.Farm time step
```

Comment:
`globale` -> `global`

Reply:
fixed in  [eaac346](https://github.com/OpenFAST/openfast/pull/2584/commits/eaac34606533b8f626e6429467196bca4b4b4736)